libtsuba/test/storage-format-version/v6-optional-datastructure-rdk.h

```diff
@@ -0,0 +1,120 @@
+#include <cstdint>
```

Comment:
why the version in the name? Why the test under "storage-format-version" as opposed to "optional-datastructures"?

Reply:
The style that is established is for tests of the storage format versions to be in the storage format version directory. More would be here, but some have to be in libgraph for dependency reasons (thats a long story).

> why the version in the name
optional datastructures were added in storage format version 6, its nice to be able to look at the test files and see what storage format version we are actually testing.